lib/controllers/tips.js

```diff
@@ -0,0 +1,34 @@
+const { Router } = require('express');
+const Tip = require('../models/Tip');
+
+module.exports = Router()
+  .post('/', (req, res, next) => {
+    Tip
+      .insert(req.body)
+      .then(tip => res.send(tip))
+      .catch(next);
+  })
+  .get('/', (req, res, next) => {
+    Tip
+      .find()
+      .then(tips => res.send(tips))
+      .catch(next);
+  })
+  .get('/random', (req, res, next) => {
+    Tip
+      .getRandom()
+      .then(tips => res.send(tips))
```

Comment:
This looks pretty good. My only question is if this is if the data on line 20 is multiple tips (as indicated above in plural) or a just single tip?